lib/pryin.ex

```diff
@@ -141,4 +139,30 @@ defmodule PryIn do
       result
     end
   end
+
+
+  @doc """
+  Add the current value for a metric.
+
+  This is independent of a running trace.
+
+  The first argument is a (string) label that can be chosen freely.
+
+  The second argument is the actual value.
+
+  Possible keys for opts are currently only `:context`,
+  which is a map of additional metadata.
+
+  Example:
+
+  ```
+  PryIn.track_metric("genserver state length", length(state.some_list), context: %{some_key: "some_value"})
+  ```
+  """
+  def track_metric(label, value, opts \\ %{}) when is_binary(label) and is_number(value) do
+    PryIn.MetricValueStore.add_metric_value(label,
+      value,
+      DateTime.to_unix(DateTime.utc_now , :milliseconds),
```

Comment:
I think you can use `PryIn.TimeHelper.utc_unix_datetime`, isn't it?

Reply:
I thought about that, too. But the helper returns microseconds and here we need milliseconds.